contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -143,9 +123,10 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
             // In the wrapped token case, we know that the bridgeToken to be burned
             // is not a "fee/burn on transfer" token, since it was deployed by this
             // contract itself.
-            if (totalAmount <= primaryFeeAmount + secondaryFeeAmount) {
-                revert InsufficientTotalAmount(totalAmount, primaryFeeAmount + secondaryFeeAmount);
-            }
+            require(
+                totalAmount >= primaryFeeAmount + secondaryFeeAmount,
```

Comment:
I think this should be `>` not `>=`

Reply:
is this to avoid us calling `encodeMintBridgeTokensData` with 0 bridge amount? Should we add a check in `mint` that the amount != 0 if this is the case?

Reply:
The check in `mint` would be in the transaction receiving the Teleporter message, rather than the one sending it here. The idea being that the amount bridged excluding fees must be non-zero.